apertium-cat.cat.dix

```diff
@@ -5121,6 +5121,13 @@
 
 </pardef>
 
+<pardef n="semid/éu__n">
+  <e>       <p><l>éu</l>         <r>u<s n="n"/><s n="m"/><s n="sg"/></r></p></e>
```

Comment:
remove "éu" on the right: `<r>éu<s n="n"/> ... ` (in the four entries).

Reply:
What should I put instead of that?

Reply:
Exactly this: 
` <e>       <p><l>éu</l>         <r>éu<s n="n"/><s n="m"/><s n="sg"/></r></p></e>`
The right part must be the same string in the paradigm name semid/**éu**__n